dvc/commands/plots.py

```diff
@@ -223,31 +223,22 @@ def run(self):
 
 
 class CmdPlotsTemplates(CmdBase):
-    TEMPLATES_CHOICES = [
-        "simple",
-        "linear",
-        "confusion",
-        "confusion_normalized",
-        "scatter",
-        "smooth",
-        "bar_horizontal_sorted",
-        "bar_horizontal",
-    ]
-
     def run(self):
-        from dvc_render.vega_templates import dump_templates
+        from dvc.exceptions import InvalidArgumentError
+        from dvc_render.vega_templates import TEMPLATES
 
         try:
-            out = (
-                os.path.join(os.getcwd(), self.args.out)
-                if self.args.out
-                else self.repo.plots.templates_dir
-            )
+            target = self.args.template
+            if target:
+                for template in TEMPLATES:
+                    if target == template.DEFAULT_NAME:
+                        ui.write(template.DEFAULT_CONTENT)
```

Comment:
```suggestion
                        ui.write(json.dumps(template.DEFAULT_CONTENT))
```

If we just provide the dict, printed string will wrap fields in single quote which standard json library wont accept.
For example:
```
dvc plots templates confusion | python -m json.tool
```
will fail